azurerm/internal/provider/required_resource_providers.go

```diff
@@ -62,6 +62,7 @@ func RequiredResourceProviders() map[string]struct{} {
 		"Microsoft.ServiceFabric":        {},
 		"Microsoft.Sql":                  {},
 		"Microsoft.Storage":              {},
+		"Microsoft.StorageCache":         {},
```

Comment:
@katbyte i've updated the `RequiredResourceProviders` func here. let me know if this is correct. thanks!